README.rst

```diff
@@ -84,17 +90,37 @@ Launch Sphinx in an ephemeral pex environment using the Sphinx entry point ``sph
 
     $ pex sphinx -e sphinx:main -- --help
 
-Build a standalone pex binary into ``pex.pex`` using the ``pex`` console_scripts entry point:
+Using ``setup.py`` Entry Points
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you have a ``pex_entry`` entry point in the `console_scripts <https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point>`_ section of your ``setup.py``:
```

Comment:
I'm not sure about the extra detail of the entry_points snippet. It's true it links to the pypa - so how could that be wrong! But, the world is already shifting https://www.python.org/dev/peps/pep-0621/#entry-points

I'm not hard against the extra detail, but hopefully its clear this just might make things _more_ confusing for, say, someone new to Python that's only used Poetry from day 1 for example.

Reply:
Hmm, that's a good point. Would it help if I added a note that Flit, Poetry, and other tools handle specifying entry points differently?

As much as I hope the world moves to using PEP-621/`pyproject.toml` as fast as possible, I still see a lot of `setup.py` files out there. And I would like to include at least _one_ example and/or link to documentation.

Reply:
Eh, I removed it, but added links for easy access to the specific parts of the documentation.